analysis_driver/quality_control/relatedness.py

```diff
@@ -88,19 +95,20 @@ def get_outfile_content(self, values):
 
                 if len(list(comparison['family_ids'])) == 1:
                     if len(comparison['proband']) == 1:
-                        gel_line = [''.join(list(comparison['family_ids'])),
-                                    comparison['proband'][0],
+                        gel_line = [comparison['proband'][0],
+                                    ''.join(list(comparison['family_ids'])),
                                     'Proband',
                                     comparison['other'][0],
+                                    ''.join(list(comparison['family_ids'])),
```

Comment:
Duplicated field?

Reply:
This is a little bit superfluous but clarifies that the two individuals being compared are from the same family.